.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
```

Comment:
I fully agree that it's our duty to maintain ecosystem stability. I was just trying to point out that as this is a low-frequency repo and Travis won't be run regularly and thus PRs can end up with unrelated failures.
Luckily for the time the new project tester was running, breakage was pretty low, before this I was having lots of issues with dmd-nightly at Mir ... so I guess if we end up having troubles with this the best step forward is to add YAML-D to Martin's Project tester.